x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -128,6 +121,12 @@ const UnstyledProcessEventDot = React.memo(
     const selectedDescendantId = useSelector(selectors.uiSelectedDescendantId);
     const nodeID = processEventModel.uniquePidForProcess(event);
 
+    /**
+     * A collection of events related to the current node and statistics (e.g. counts indexed by event type)
```

Comment:
:red_circle: `ResolverNodeStats` does not contain a `collection of events related to the current node` unless I'm missing something. I'd suggest we either: 1) Change the comment here or 2) If it _does_ have a collection of events, the selector name should be something more broad like `relatedEventsAndStats`

Reply:
i'll remove the comment. fwiw, that was the existing comment for `relatedEventStatsForProcess`, which contained exactly what `relatedEventStats` now contains.

Reply:
Maybe at one point it _did_ have that collection with it in whatever form it was being consumed here.